### ISSUE TITLE
Improve GIF view

### ISSUE THREAD

**Issue by jesmrec** [owncloud/ios-app#323]
Opening a GIF, seems to be a plain view (white background, alligned to top):

![Mar-18-2019 11-03-28](https://user-images.githubusercontent.com/14894746/54522476-9ca78d00-496d-11e9-857f-e1fc10f8594b.gif)

**Comment by michaelstingl** [comment]
@jesmrec What is the behaviour after QL-merge? Close here?

**Comment by jesmrec** [comment]
After QL merge, this is in the same point (no improvements did there regarding GIFs)

**Comment by michaelstingl** [comment]
> After QL merge, this is in the same point (no improvements regarding GIFs)

Same as in screenshot from March?

**Comment by jesmrec** [comment]
right. The improvement was initially included in QL, but after some problems in implementation, it was decided not to be included. You can find the reason here: 

https://github.com/owncloud/ios-app/pull/493#issuecomment-553337699

finally the implementation was the previous one:

https://github.com/owncloud/ios-app/pull/493#issuecomment-555992449
